README.md

```diff
@@ -290,3 +291,15 @@ Then curl the metrics endpoint
 	curl localhost:8080/metrics
 
 To run the e2e tests locally see the documentation in [tests/README.md](./tests/README.md).
+
+#### Add New Kubernetes Resource
+
+Adding a new Kubernetes resource and its respective metrics requires modifying a few more files than you may think.
```

Comment:
I find this style of writing more fitting for a blog post. Lets just say something like:

```
The following steps are needed to introduce a new resource and respective resource metrics.
```

Reply:
SGTM